docs/docs/concepts/zitadel/objects/jwt_idp.md

```diff
@@ -0,0 +1,55 @@
+---
+title: JWT IDP
+---
+
+
+# JWT IDP
+
+JSON Web Token Identity Provider (JWT IDP) gives you the possibility to use an (existing) JWT as federated identity.
+Imagine you have a Web Application Firewall (WAF) which handles your session for an existing application.
+You're now creating a new application which uses ZITADEL as its IDP / Authentication Server.
+The new app might even be opened from within the existing application and you want to reuse the session information for the new application.
+This is where JWT IDP comes into place.
+
+All you need to provide is an endpoint where ZITADEL can receive a JWT and some information for its signature verification.
+
+## Authentication using JWT IDP
+
+The authentication process then might look like the following:
+
+![JWT IDP Architecture](/img/concepts/objects/jwt_idp.png)
+
+1. The user is logged into th existing application and the WAF holds the session information. It might even send a JWT to the application.
+   The new application is opened by clicking on a link in the existing application.
+2. The application bootstaps and since it cannot find a session, it will create an OIDC Authorization Request to ZITADEL.
+   In this request it provides a scope to directly request the JWT IDP.
+3. ZITADEL will do so and redirect to the preconfigured JWT Endpoint. While the endpoint is behind the WAF, ZITADEL is able to receive a JWT from a defined http header.
+   It will then validate its signature, which might require to call the configured Keys Endpoint.
+   If the signature is valid and token is not expired, ZITADEL will then use the token and the enclosed `sub` claim as if was an id_token returned from a OIDC IDP:
+   It will try to match a user's external identity and if not possible, create a new users with the information the provided token.
```

Comment:
```suggestion
   It will try to match a user's external identity and if not possible, create a new user with the information of the provided token.
```